cyences_app_for_splunk/appserver/static/visualizations/network_telemetry_map/src/network_telemetry_map.js

```diff
@@ -0,0 +1,186 @@
+define([
+    'jquery',
+    'leaflet',
+    'leaflet-curve',
+    'api/SplunkVisualizationBase',
+    'api/SplunkVisualizationUtils',
+    'splunkjs/mvc'
+],
+    function (
+        $,
+        L,
+        LCurve,
+        SplunkVisualizationBase,
+        SplunkVisualizationUtils,
+        mvc
+    ) {
+
+        return SplunkVisualizationBase.extend({
+
+            initialize: function () {
+                // Save this.$el for convenience
+                this.$el = $(this.el);
+                // Add a css selector class
+                this.$el.addClass('network_telemetry_map');
+            },
+
+            getInitialDataParams: function () {
+                return ({
+                    outputMode: SplunkVisualizationBase.ROW_MAJOR_OUTPUT_MODE,
+                    count: 50000
+                });
+            },
+
+            formatData: function (data, config) {
+                return data;
+            },
+
+            getConfig: function (property, config) {
+                return SplunkVisualizationUtils.escapeHtml(config[this.getPropertyNamespaceInfo().propertyNamespace + property]);
+            },
+
+            updateView: function (data, config) {
+
+                this.$el.empty();
+                if (this.map) {
+                    this.map.remove()
+                }
+
+                function getNode(name, props) {
+                    var ele = document.createElementNS("http://www.w3.org/2000/svg", name);
```

Comment:
Seems like this is for testing.

Reply:
As discussed It's required for svg definition